dashboard/src/components/AppUpgrade/AppUpgrade.tsx

```diff
@@ -36,8 +36,7 @@ interface IAppUpgradeProps {
   checkChart: (repo: string, chartName: string) => void;
   fetchChartVersions: (id: string) => Promise<IChartVersion[]>;
   getAppWithUpdateInfo: (releaseName: string, namespace: string) => void;
-  getChartVersion: (id: string, chartVersion: string) => void;
-  getChartValues: (id: string, chartVersion: string) => void;
+  getChartVersionWithValuesAndSchema: (id: string, chartVersion: string) => void;
```

Comment:
Just a thought: given that you're removing `getChartVersion`, could the new function just be `getChartVersion`, or does it need to be so specific?